lib/oppen/printer.rb

```diff
@@ -149,7 +149,7 @@ def handle_end(token)
         tokens[right] = token
         size[right] = -1
         scan_stack.push right
-        if config&.eager_print &&
+        if config&.eager_print? &&
            (!scan_stack.empty? && right_total - left_total < print_stack.space)
```

Comment:
I don't understand the parens.
```ruby
        if config&.eager_print? \
          && !scan_stack.empty? \
          && right_total - left_total < print_stack.space

Reply:
I like the parens to make it clear that the first condition is testing if we have the option activated while the last two are here to check the necessity of using the custumization. I they bother you, I can remove them